apps/.eslintrc.js

```diff
@@ -352,9 +355,6 @@ module.exports = {
         'test/util/tutorialExplorer/**',
         'test/util/weblab/**',
       ],
-      rules: {
-        'import/order': 'off',
-      },
```

Comment:
This is a readability change (ironically breaking alphabetical order 😆 ). The list of excluded dirs is very long, and you have to scan to the very end to find out WHICH rule has all these excluded dirs.